app/views/forums/_forum.html.erb

```diff
@@ -1,7 +1,7 @@
 <div id="<%= dom_id forum %>">
   <p>
-    <strong>Forum name:</strong>
-    <%= forum.forum_name %>
+    <strong>Forum description:</strong>
+    <%= forum.description %>
```

Comment:
I see that you've made changes based on [my comment](https://github.com/melissezeigler/R7-forum/pull/1/#discussion_r1702887557) from last time, but you probably want to keep both of these fields, not swap one for the other.